src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1596,6 +1596,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("PING", 0.1f);
 		configure_stream_local("POSITION_TARGET_GLOBAL_INT", 1.0f);
 		configure_stream_local("POSITION_TARGET_LOCAL_NED", 1.5f);
+		configure_stream_local("RAW_RPM", 2.0f);
```

Comment:
Can you please add as well to `MAVLINK_MODE_ONBOARD` with an higher rate?

Reply:
Message added to `MAVLINK_MODE_ONBOARD` with 5 Hz rate. It should be enough.